### PR TITLE
Add debug tutorial reference crictl

### PR DESCRIPTION
**Pull request by daxmc99**
Fixes https://github.com/sourcegraph/infrastructure/pull/2267#issuecomment-735920328 

Part of sourcegraph/sourcegraph#15194

### CODE REVIEW

File: handbook/engineering/deployments/debugging/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+This guide will allow you to debug one of our running services in the **dogfood** deployment which mimics our customer environment. All of our services here run on a Kubernetes cluster.
+
+- [Debugging](#debugging)
+----[Gain-access-to-the-cluster](./tutorial.md#gain-access-to-the-cluster)
+
+## Intro to crictl
+
+_[Kubernetes Docs](https://kubernetes.io/docs/tasks/debug-application-cluster/crictl/)_
+
+When debugging pods on the node itself you should prefer `critctl` from <https://github.com/kubernetes-sigs/cri-tools/releases> over docker.
```
Comment (daxmc99) [inline handbook/engineering/deployments/debugging/index.md:14]: I think this the biggest change that comes out of this. `crictl` is mostly compatible with docker.

File: handbook/engineering/deployments/debugging/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Debugging
+
+In order to debug the services we deploy you should understand the environment they are deployed on.
+
+This guide will allow you to debug one of our running services in the **dogfood** deployment which mimics our customer environment. All of our services here run on a Kubernetes cluster.
+
+- [Debugging](#debugging)
+----[Gain-access-to-the-cluster](./tutorial.md#gain-access-to-the-cluster)
```
Comment (bobheadxi) [inline handbook/engineering/deployments/debugging/index.md:8]:
```suggestion
  - [Gain-access-to-the-cluster](./tutorial.md#gain-access-to-the-cluster)
```

(this doesnt seem to render in github?)

File: handbook/engineering/deployments/index.md [hunk 3]
```diff
@@ -5,20 +5,27 @@ We maintain multiple [instances](instances.md) of Sourcegraph:
...
-- [Kubernetes](./kubernetes.md): setting up access, tips, and more
-- [Security](./security.md): TLS configuration,
-- [Testing](./testing.md): deploying test instances of Sourcegraph
-- [Playbooks](./playbooks.md): how-to guides for common tasks
-- [deploy-sourcegraph](#deploy-sourcegraph): base configuration for Kubernetes Sourcegraph deployments
-- [PostgresSQL](./postgresql.md): Tips for working with Postgres and our deployments
+- [Deployments](#deployments)
+  - [Deployment basics](#deployment-basics)
+    - [Kubernetes](./kubernetes.md): setting up access, tips, and more
+    - [Security](./security.md): TLS configuration,
```
Comment (bobheadxi) [inline handbook/engineering/deployments/index.md:17]: I think we should omit the top-level header here ("Deployments"), but I'm guessing this is VSCode trying to do its auto-TOC thing 😓

File: handbook/engineering/deployments/debugging/index.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Debugging
+
+In order to debug the services we deploy you should understand the environment they are deployed on.
+
+This guide will allow you to debug one of our running services in the **dogfood** deployment which mimics our customer environment. All of our services here run on a Kubernetes cluster.
```
Comment (bobheadxi) [inline handbook/engineering/deployments/debugging/index.md:5]: Link to the k8s entry in the instances doc maybe?

File: handbook/engineering/deployments/index.md (hunk 3 above)
Comment (daxmc99) [reply handbook/engineering/deployments/index.md:17]: This is the auto TOC thing 😬 I can drop this top-level header

File: handbook/engineering/deployments/debugging/index.md (hunk 4 above)
Comment (daxmc99) [reply handbook/engineering/deployments/debugging/index.md:5]: Good idea

File: handbook/engineering/deployments/index.md [hunk 5]
```diff
@@ -5,20 +5,26 @@ We maintain multiple [instances](instances.md) of Sourcegraph:
 - [Instances](./instances.md): information about our different instances
   - [sourcegraph.com](instances.md#sourcegraph-com) is our production deployment for open source code.
   - [sourcegraph.sgdev.org](instances.md#sourcegraph-sgdev-org) is our private deployment of Sourcegraph that contains some of our private code.
-  - [k8s.sgdev.org](instances.md#k8s-sgdev-org) is a dogfood deployment that replicates the scale of our largest customers. Note that this also contains all of our private code.
-- [Managed instances](../distribution/managed/index.md) are deployments of Sourcegraph we manage for customers.
-  - [demo.sourcegraph.com](instances.md#demo-sourcegraph-copm) is a managed instance used for CE demos.
+  - [k8s.sgdev.org](instances.md#k8s-sgdev-org) is a dogfood deployment that replicates the scale of our largest customers. Note that this also contains all of our private code. - [Managed instances](../distribution/managed/index.md) are deployments of Sourcegraph we manage for customers.
```
Comment (pecigonzalo) [inline handbook/engineering/deployments/index.md:8]: This seems to be an error.

**Review by pecigonzalo — APPROVED** [review]

**Pull request merged**